<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ResourceFile.qrc">
    <normaloff>:/Image/Resource/touch_fish.png</normaloff>:/Image/Resource/touch_fish.png</iconset>
  </property>
  <widget class="QPushButton" name="apply_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="descript">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>参数文件位于根目录/Config.ini，删除文件即可重置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>90</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="start_h_input">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="start_m_input"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>分</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>月薪</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="month_salary_input"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>结束时间 </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>开始时间 </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="end_h_input"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="end_m_input"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>分</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="developer_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:right;
border-style:outset;
border-width:0px;
color:rgb(111, 111, 111);
text-decoration:underline;</string>
   </property>
   <property name="text">
    <string>by 油灰大狼</string>
   </property>
  </widget>
  <zorder>developer_button</zorder>
  <zorder>apply_button</zorder>
  <zorder>cancel_button</zorder>
  <zorder>descript</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources>
  <include location="ResourceFile.qrc"/>
 </resources>
 <connections/>
</ui>
